libs/SalesforceSDKCommon/SalesforceSDKCommon/Classes/Keychain/KeychainHelper.swift

```diff
@@ -417,118 +400,6 @@ public class KeychainHelper: NSObject {
             }
         }
     }
-    
```

Comment:
Removing that code means that apps won't be upgrade from 9.0 to 13.0 - but I don't think it would work anyway.

Reply:
this will be nice not to have in the logs anymore